src/visualization/Turi Create Visualization/src/VegaContainer.swift

```diff
@@ -271,6 +287,22 @@ class VegaContainer: NSObject, WKScriptMessageHandler {
             }
         }
     }
+    
+    public func get_image(completion: @escaping (NSImage) -> Void) {
+        let jsString = String(format: "export_png();");
```

Comment:
This could just be `let jsString = "export_png();"`, right?

Reply:
Yes, your absolutely correct. I changed it.